mycpp/control_flow_pass.py

```diff
@@ -514,6 +548,27 @@ def visit_call_expr(self, o: 'mypy.nodes.CallExpr') -> T:
                     self.current_statement_id,
                     pass_state.FunctionCall(join_name(full_callee, delim='.')))
 
+                callee_t = self.types.get(o.callee)
```

Comment:
I was actually able to remove this (and simplify this region) by having the `Bind` fact simply refer to the position of the argument being passed instead of the parameter name from the callee's definition. If we end up needing the parameter names in the future, I will reintroduce this logic as a helper function and refactor it to make it more obvious what it's doing.